src/role-access/services/role-exclusions.service.ts

```diff
@@ -1,7 +1,7 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { RoleExclusion } from '../models/role-exclusions/role-exclusion.model';
+import { RoleExclusion } from '../models/role-exclusion.model';
 
 @Injectable()
 export class RoleExclusionsService {
```

Comment:
Can the generic type <any> be replaced with an actual type?

Reply:
Done

Reply:
Great. Thanks.